packages/react-devtools-shared/src/backend/utils.js

```diff
@@ -183,3 +183,10 @@ export function format(
 
   return '' + formatted;
 }
+
+export function isSynchronousXHRSupported() {
+  return (
+    window.document.featurePolicy &&
```

Comment:
I think `window.document == null` for standalone DevTools, so this could cause an error if we were to ever try enabling reload-and-profile there (which I intend to do as soon as the backend supports it). Can we make this check safer?

Reply:
Have added a check for `window.document` being truthy

Reply:
Awesome!

Were you using a specific URL to test the disabled state? (I don't actually know of a website that uses this new header.)

Reply:
Fair enough 😆 I guess I was being lazy. Thanks!